chainermn/communicators/__init__.py

```diff
@@ -17,11 +17,13 @@ def create_communicator(
     two_dimensional     OK  Required Each node has multiple NICs or HCAs
     single_node         OK  Required Single node with multiple GPUs
     flat                OK           N/A
+    nccl                OK  Required N/A
```

Comment:
Update: `nccl` is recommended when NCCL2 is available in the environment, but it's still experimental support.